src/components/dataList.js

```diff
@@ -118,10 +118,24 @@
           repeat();
         });
 
+        React.useEffect(() => {
+          if (isDev) return;
+          if (options.currentRecord !== '') {
+            B.triggerEvent('OnRowClick', options.currentRecord);
```

Comment:
Do we need a `useEffect` for this? Is it possible to add this to the `handleClick` function?

Reply:
We should always use a `useEffect` because setingState is not async. Before hooks this was acheived by passing a second argument to `setState(mutation, callback)`. But with `useState()` useEffects are the best way to perform actions "after" state has changed.